pkgs/development/misc/xtensa/newlib.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, texinfo, flex, bison, fetchFromGitHub, crossLibcStdenv, buildPackages, espressifXtensaOverlays }:
+
+crossLibcStdenv.mkDerivation {
+  name = "newlib";
```

Comment:
```suggestion
  pname = "newlib";
  version = "...";
```

Reply:
This is built from custom sources, so a version number would not be very informative.

Reply:
Every mkDerivation should have pname+version so that at some point we can rely on this. Maybe set the version to ``custom-esp2020r3``?